modules/machine-delete.adoc

```diff
@@ -9,11 +9,11 @@
 
 You can delete a specific machine. 
 
-//todo: confirm if this statement is still accurate w/CPMS
 [NOTE]
 ====
 You cannot delete a control plane machine.
-====  
+==== 
+//todo: assess for CPMS impact
```

Comment:
Machine management > Deleting a machine > [Deleting a specific machine](https://53397--docspreview.netlify.app/openshift-enterprise/latest/machine_management/deleting-machine.html#machine-delete_deleting-machine)

Reply:
I'm not really sure what this note is intending to convey. Does it mean you can't delete one permanently? That is still true afaik. 

But now you can delete one and have the CPMS make a new one to replace it (for supported platforms ofc). Is this the appropriate clarification to make?

Reply:
I think really it should have said "You should not delete a control plane machine". Of course you can delete it, but it might have catastrophic results.

Having a note suggesting that control plane machines should not be deleted because this might cause cluster failures seems reasonable

Reply:
Any CPMS strat works here, if they delete the Machine and there's a CPMS, it will recreate it